<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateUser</class>
 <widget class="QDialog" name="CreateUser">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>172</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>172</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create a new user</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Logo.ico</normaloff>../Logo.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="edtUsername">
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtCPword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblWarning">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>Passwords do not match!</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>Create user</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
